tgui-next/packages/tgui/interfaces/OreBox.js

```diff
@@ -0,0 +1,48 @@
+import { act } from '../byond';
+import { Fragment } from 'inferno';
+import { BlockQuote, Box, Table, Button, Section } from '../components';
+import { toTitleCase } from 'common/string';
+
+export const OreBox = props => {
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
+  const { materials } = data;
+  return (
+    <Fragment>
+      <Section
+        title="Ore Box"
+      >
+        <BlockQuote mb={1}>
```

Comment:
Remove the block quote? It looks weird.

Reply:
Looks fine to me.

Reply:
Just a suggestion. Maybe it's because of the text under it that's throwing me off.